components/task-form.jsx

```diff
@@ -0,0 +1,112 @@
+'use client'
+
+import React, { useState } from 'react';
+import DatePicker from 'react-datepicker';
+import 'react-datepicker/dist/react-datepicker.css';
+import GrayButton from './gray-button';
+
+const TaskForm = () => {
+  const [deadline, setDeadline] = useState(null);
+  const [sender, setSender] = useState('');
+  const [documentType, setDocumentType] = useState('');
+  const [urgency, setUrgency] = useState('');
+  const [receiver, setReceiver] = useState('');
+  const [additionalInfo, setAdditionalInfo] = useState('');
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+
+    // You can handle the form submission logic here
+    const formData = {
```

Comment:
If you add `name` attributes to all your inputs and you're not going to be manually updating their values (only the user will do that by using the form), you don't need to have separate variables for each one of them:

```suggestion
    const formData = Object.fromEntries(new FormData(e.target));
```

`formData` will return an object with all your form values.

More info [here](https://stackabuse.com/convert-form-data-to-javascript-object/)

Reply:
Just made a commit to add the recommended change, also eliminating redundant code or related to the previous functionality.